### PR TITLE
Front end work v3

### PR DESCRIPTION
**Pull request by masoncpott**
@danielharmon @gigicupp I would greatly appreciate a code review for this.

I basically rebuilt each product card displayed in the slider container.  This time using css-grid and fussing with the images more. The issue we were having with the first few products being too large on the page should now be resolved.

My next task is to implement some network requests on the back end to get all the live data but I would like to merge this progress into master at this point.

### CODE REVIEW

File: client/src/components/SliderCard.jsx [hunk 1]
```diff
@@ -3,21 +3,55 @@ import ReactDOM from 'react-dom';
...
-  sizes: 723px;
+  object-fit: cover;
 `;
 
-const StyledText = styled.div`
-  display: flex;
-  justify-content: flex-start;
-  padding: 0 1.6rem 0;
+const StyleImg = styled.img`
+  max-width: auto;
```
Comment (gigicupp) [inline client/src/components/SliderCard.jsx:32]: @masoncpott I have never used this before. What's the difference between max-width and width?

File: client/src/components/SliderCard.jsx [hunk 2]
```diff
@@ -3,21 +3,55 @@ import ReactDOM from 'react-dom';
 import styled from 'styled-components';
 
 const StyledCard = styled.div`
+  flex: 1 0 25%;
+  display: grid;
```
Comment (gigicupp) [inline client/src/components/SliderCard.jsx:7]: @masoncpott grid is definitely my new found passion lol!

File: client/src/components/SliderCard.jsx [hunk 3]
```diff
@@ -31,10 +65,12 @@ class SliderCard extends React.Component {
     return (
       <>
       <StyledCard>
-        <img src={this.props.image[0]}></img>
-        <StyledText>{this.props.title}</StyledText>
-        <StyledText>{this.props.inventory.price}</StyledText>
-        <StyledText>icons</StyledText>
+        <CardPictureStyle >
```
Comment (gigicupp) [inline client/src/components/SliderCard.jsx:68]: could use de-structuring to make it less wordy.

File: client/src/SimilarProductsComponent.jsx [hunk 4]
```diff
@@ -21,15 +22,21 @@ const SliderContainer = styled.div`
...
+  border-color: transparent;
   border-radius: 50%;
+  padding: 5px;
   background-color: white;
   color: black;
   font-family: Nunito Sans;
   font-weight: 700;
   font-size: 1.2rem;
-  align-items: center;
+  align-self: center;
```
Comment (gigicupp) [inline client/src/SimilarProductsComponent.jsx:33]: ooh! I will look into what this does. haven't used it yet.

**Review by gigicupp — COMMENTED** [review]
@masoncpott looks good! you are good to go!

File: client/src/components/SliderCard.jsx (hunk 1 above)
Comment (masoncpott) [reply client/src/components/SliderCard.jsx:32]: I think max-width helps with resizing when a window gets smaller?

File: client/src/components/SliderCard.jsx (hunk 3 above)
Comment (masoncpott) [reply client/src/components/SliderCard.jsx:68]: Yes, this will change when I start incorporating live data from the API calls it makes on the back end

**Pull request merged**